c2corg_ui/templates/profile/data.html

```diff
@@ -14,46 +14,48 @@
 
 % else:
 
-  % if geometry:
-
-    % if profile.get('areas'):
-      <article class="location-static">
-        ${show_areas(profile)}
-      </article>
+  <div app-view-details>
+    % if geometry:
+
+      % if profile.get('areas'):
+        <article class="location-static">
+          ${show_areas(profile)}
+        </article>
+      % endif
+
+      <div ng-init='profileFeature = {
+          "type": "FeatureCollection",
+          "properties": {},
```

Comment:
That's not very clear. At least for ``Feature`` it seems required:
http://geojson.org/geojson-spec.html#feature-objects
>  A feature object must have a member with the name "properties".

I have found no indication for ``FeatureCollection``. But that doesn't hurt and it's consistent with the FeatureCollection of the other document types.